Simon-Initiative/course-digest
# [FEATURE] Add custom dnd migration support

This PR adds support for migrating Custom Drag and Drop questions.  

The main work here is:
1. The detection of this type of question, by looking for the custom javascript tag.
2. Conversion of the underlying mult_input question using the existing multi_input conversion code
3. Reading and conversion of the external layout file, moving the targetArea, initiators and layoutStyles into the activity content.
4. Detection of "non direct" image references within the layout styles, and addition of them into the media library, and replacement of their references with the full URL that the fill will exist as in Torus. 

To test this out, end to end:
1. Run Torus with the corresponding `custom-dnd` branch.
2. Generate a course digest of the test course that is embedded in this repo.
3. Ingest that digest into Torus.
4. Verify that you can preview, deliver and edit the two custom DnD questions that appear in a page in that course.
